go.mod

```diff
@@ -1,6 +1,8 @@
 module github.com/sapcc/absent-metrics-operator
 
-go 1.22
+go 1.22.0
```

Comment:
Newer go versions demand this to contain the .0 if any dependency includes it, which is k8s in our case, which is rather bad but it it is what it is. I've fixed it on master.

Reply:
I see. However, the CI still fails.

Let's keep this PR open until CI is green again. If you have any fixes kindly add them to this PR's branch instead of master.

Reply:
#186 somehow didn't update the peer dependencies correctly. Not idea why though, that shouldn't happen like that without merge conflicts or so.

Reply:
and because of the wide gitignore file, we didn't commit all requires files, so the runner is missing them but they exist locally. Classic one.

Reply:
Good to know about gitignore's greedy matching. 

Thanks for the fix 🙌🏾